sqlalchemy_history/builder.py

```diff
@@ -200,3 +203,17 @@ def create_column_aliases(self, version_classes):
                         continue
 
                     version_class_mapper.add_property(key, sa.orm.column_property(version_class_column))
+
+    def create_association_proxies(self, version_classes):
+        """
+        Create Association proxy for Column of Versioned Models from Original Model
+        """
+        for cls in version_classes:
+            assoc_prox_maps = get_association_proxy_mapping(cls)
+            for attr, proxy in assoc_prox_maps.items():
+                versioned_target_class = version_class(cls)
+                setattr(
+                    versioned_target_class,
+                    attr,
+                    property(fget=proxy.__get__, fset=proxy.__set__, fdel=proxy.__delete__),
```

Comment:
Why do we need fset and fdel ? I would assume this should be for viewing information only